lib/jekyll-feed/generator.rb

```diff
@@ -136,5 +147,12 @@ def normalize_posts_meta(hash)
       config["path"] ||= hash["posts"]["path"]
       hash
     end
+
+    # Returns the configured title for the collection if defined. Otherwise, it
+    # returns a fallback value of "Site Name | Collection"
+    def title(collection = nil)
```

Comment:
Why is the `collection` parameter optional here?
When is `data_for_collection(nil)` from the next line ever valid?